conda_build/build.py

```diff
@@ -592,7 +592,7 @@ def test(m, move_broken=True):
     specs = ['%s %s %s' % (m.name(), m.version(), m.build_id())]
 
     # add packages listed in test/requires
-    specs += m.get_value('test/requires', [])
+    specs.extend(ms.spec for ms in m.ms_depends('run'))
 
```

Comment:
Yeah, I hear you. `requirements/build` and `requirements/run` I get. And I understand that testing might require packages _in addition_ to the run requirements. But why not `requirements/test`?